src/installer/pkg/archives/dotnet-nethost.proj

```diff
@@ -0,0 +1,36 @@
+<Project Sdk="Microsoft.Build.NoTargets">
+  <PropertyGroup>
+    <CreateSymbolsArchive>true</CreateSymbolsArchive>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.DotNet.Build.Tasks.Archives" Version="$(MicrosoftDotNetBuildTasksArchivesVersion)" />
+  </ItemGroup>
+
+
+  <Target Name="PublishToDisk">
+    <Error Condition="'$(OutputPath)' == ''" Text="Publishing to disk requires the OutputPath to be set to the root of the path to write to." />
+
+    <ItemGroup>
+      <FilesToPublish Include="$(DotNetHostBinDir)\$(LibPrefix)nethost$(LibSuffix)"
+            Destination="$(OutputPath)$(LibPrefix)nethost$(LibSuffix)" />
+      <FilesToPublish Include="$(DotNetHostBinDir)\$(LibPrefix)libnethost$(StaticLibSuffix)"
+            Destination="$(OutputPath)$(LibPrefix)libnethost$(StaticLibSuffix)" />
```

Comment:
```suggestion
            Destination="$(OutputPath)$(LibPrefix)nethost$(StaticLibSuffix)" />
```

Previously in 3.1 this was just 

```
      <NetHostFiles Include="$(_NetHostOutputDir)nethost.h" />
      <NetHostFiles Include="$(_NetHostOutputDir)$(LibPrefix)nethost$(LibSuffix)" />
      <NetHostFiles Condition="'$(OSGroup)' == 'Windows_NT'" Include="$(_NetHostOutputDir)nethost.lib" />
```

Is there both a lib and a static lib?

Reply:
Yes, there is both. The static lib was added in 5.0. It should be `libnethost.a` on Unix and `libnethost.lib` on Windows, so I think this should be:
```suggestion
            Destination="$(OutputPath)$(StaticLibPrefix)nethost$(StaticLibSuffix)" />
```